### ISSUE TITLE
'list_messages()' and 'get_message()' functions only return response status code

### ISSUE THREAD

**Issue by adithya249** [GearPlug/microsoftgraph-python#42]
Calling 'client.mail.list_messages()' and 'client.mail.get_message({message_id})' end up returning only the response status code and not what they're intended to return; i.e., a list of messages  and the contents of a message. 

![Screenshot from 2022-10-23 16-36-40](https://user-images.githubusercontent.com/109500304/197388614-fc9d1bfc-714b-43d3-82f3-9dad9002ed34.png)

Also tried fetching the json of the response object 'msgs', but it resulted in an attribute error which stated that the response object has no attribute 'json'.

**Comment by shifqu** [comment]
Since you are in repl, if you just type the name of the variable it will actually call `print(msgs)` or `str(msgs)`.

You could try to see which attributes are on the `msgs` variable by calling `dir(msgs)`.

I checked the code in this repository and noticed that `list_messages` actually returns `self._client._get(...)`, which in turn returns a `Response`. This response can be found [here](https://github.com/GearPlug/microsoftgraph-python/blob/master/microsoftgraph/response.py)

So basically, to get the json data of your messages, you will want to call `msgs.data`.
You can also get the original response by accessing the attribute `original` on the `Response`
